src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -1048,6 +1049,58 @@ describe('Form Model', () => {
         expect(actual).toEqual(expected)
       })
     })
+
+    describe('deleteFormLogic', () => {
+      const logicId = new ObjectId().toHexString()
+      const mockFormLogic = {
+        form_logics: [
+          {
+            _id: logicId,
+            id: logicId,
+          } as ILogicSchema,
+        ],
+      }
+
+      it('should return form upon for successful delete', async () => {
```

Comment:
can we test deleting from an array containing multiple logic fields as well pls?

Reply:
added